src/components/TodoItem.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import * as Action from '../api/todos';
```

Comment:
```suggestion
import * as action from '../api/todos';
```
camelCase for common variables (PascalCase for classes, components, types)